ui_framework/src/global_styling/reset/_reset.scss

```diff
@@ -52,13 +53,13 @@ body {
 
 a {
   text-decoration: none;
+
   &:hover {
-    text-decoration: underline;
+    text-decoration: none;
   }
 
   &:focus {
-    text-decoration: underline;
-    background: $kuiFocusBackgroundColor;
+    text-decoration: none;
```

Comment:
The reason I made this change was because `KeyPadMenuItemButton` wasn't picking up the underline on hover, since it's a `<button>` and not a link. This was a really subtle difference and I can see it being very easily overlooked. Imagine someone changing the behavior of the link sometime in the future -- how will we know which components need to be updated to share this behavior?

What we really need is a single source of truth for link-like behavior, which can be consumed by all the components which need it, and which will be the one place we need to update when we want to change this behavior.

I think we should create a mixin to encapsulate this behavior and solve this problem. What do you think?